queue_job/models/base.py

```diff
@@ -216,3 +216,29 @@ def _job_store_values(self, job):
         :return: dictionary for setting job values.
         """
         return {}
+
+    @api.model
+    def _job_prepare_context_before_enqueue_keys(self):
+        """Keys to keep in context of stored jobs
+
+        Empty by default for backward compatibility.
+        """
+        return {
+            "tz",
+            "lang",
+            "uid",
+            "allowed_company_ids",
+            "force_company",
+            "active_test",
+        }
```

Comment:
```suggestion
        return (
            "tz",
            "lang",
            "uid",
            "allowed_company_ids",
            "force_company",
            "active_test",
        )
```

Reply:
Of course, I'm starting to lose my python skills 😱